includes/class-wp-job-manager-email-notifications.php

```diff
@@ -340,29 +325,88 @@ public static function output_header( $email, $sent_to_admin, $plain_text = fals
 	/**
 	 * Output email footer.
 	 *
-	 * @param WP_Job_Manager_Email $email          Email object for the notification.
-	 * @param bool                 $sent_to_admin  True if this is being sent to an administrator.
-	 * @param bool                 $plain_text     True if the email is being sent as plain text.
+	 * @param string $email_notification_key  Email object for the notification.
+	 * @param bool   $sent_to_admin           True if this is being sent to an administrator.
+	 * @param bool   $plain_text              True if the email is being sent as plain text.
 	 */
-	public static function output_footer( $email, $sent_to_admin, $plain_text = false ) {
-		$template_segment = locate_job_manager_template( self::get_template_file_name( 'email-footer', $plain_text ) );
-		if ( ! file_exists( $template_segment ) ) {
+	public static function output_footer( $email_notification_key, $sent_to_admin, $plain_text = false ) {
+		$template_segment = self::email_template_path_alternative( $email_notification_key, 'email-footer', $plain_text );
+		if ( false === $template_segment ) {
+			$template_segment = self::locate_template_file( 'email-footer', $plain_text );
+		}
+		if ( ! $template_segment || ! file_exists( $template_segment ) ) {
 			return;
 		}
 		include $template_segment;
 	}
 
 	/**
-	 * Add email notification settings.
+	 * Checks for an alternative email template segment in the template path specified by the current email.
+	 * Useful to provide alternative email headers and footers for a specific WPJM extension plugin.
+	 *
+	 * @param string  $email_notification_key  Email object for the notification.
```

Comment:
`s/object/key`

Reply:
Fixed in 6dd46dd2d7c431b867df4ba281e3e9a1f2bc1277